StripeUICore/StripeUICore/Source/Elements/TextField/TextFieldElement.swift

```diff
@@ -90,7 +90,9 @@ import UIKit
             attributedText: configuration.makeDisplayText(for: text),
             keyboardProperties: configuration.keyboardProperties(for: text),
             validationState: configuration.validate(text: text, isOptional: configuration.isOptional),
-            logo: configuration.logo(for: text),
+            accessoryViewBuilder: { () -> UIView? in
+                self.configuration.accessoryView(for: self.text, theme: self.theme)
+            },
```

Comment:
I don't _love_ this solution, but is the best I could think of.
The other options were:

-  Rebuilding the viewModel, which may have some unintended consequences and I would have to do some extra digging to make sure where would be the best place to do that.
- Use a similar lightMode/darkMode approach as before, but I don't like that either.
- One of the suggestions in the original PR, returning an array of lightMode/darkMode logos, and make the TextFieldElement handle the rotation, this is probably my least favorite, I'd rather push the responsibility as far up the chain as possible.

I'm open to other suggestions.

Reply:
It's so we can get a view that's configured with the current theme.
The issue with the CVV regression was that it was always a light mode view, and that was because the `accessoryView` was created with a light theme, when the traits changed the `accessoryView` in the `TextFieldView` is updated, but the the `viewModel` still contains the same view which was created in light mode, by using a closure a new view is created which now uses the dark theme.

Reply:
Ah hm, can you not override `traitCollectionDidChange` in the view for that?

Also, following on my comment, now I'm wondering - _does_ this re-create the view often?  Like, if you breakpoint and start typing in the field...does it get re-created every keypress?

Reply:
> Also, following on my comment, now I'm wondering - does this re-create the view often? Like, if you breakpoint and start typing in the field...does it get re-created every keypress?

Well, for the Credit Card field is working as intended, the `RotatingCardBrandsView` is cached and only updated as necessary every time the text changes.
The CVC field does get recreated every time the CVC text changes, so it would either need to cache the view as well, or use a different approach.

> Ah hm, can you not override traitCollectionDidChange in the view for that?

It is already doing that, but since the `viewModel` doesn't change the view is not updated. So one alternative would be to recreate the `viewModel` in `traitCollectionDidChange`, my only concern is whether that's the only place where the viewModel needs to be recreated.